sources/platform/console/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Apify Console
+description: Learn about Apify Console's easy account creation and user-friendly homepage for efficient web scraping management.
+sidebar_position: 1
+category: platform
+slug: /console
+---
+
+**Learn about Apify Console's easy account creation and user-friendly homepage for efficient web scraping management.**
+
+---
+
+## Sign Up
+
+To use Apify Console, you need to create Apify account. [Sign up here](https://console.apify.com/sign-up).
+
+![apify-console-sign-up](./images/console-signup.png)
+
+## Log In
+
+To log in to Apify Console, you can use one of the following methods:
+
+* Google Single Sign-On
+* GitHub Single Sign-On
+* The email and password set during account creation
```

Comment:
Suggestion: Maybe I'd move this to the signup part

Reply:
hmm it might warrant to have them in both parts, since you can use them to sign up and log in respectively

Reply:
added to sing up section as well

Reply:
To me that would be one of the "way too verbose" things. It's quite usual that if you sign up using google, you can also log in with it later.

Reply:
hmm I could also just remove it all together, we have screenshots there that show all the possible ways of signing up and logging in

Reply:
Yeah, true.

Reply:
I think this is the way to go, I will remove it from both sections and rely for now on screenshots.